x-pack/docs/en/security/auditing/event-types.asciidoc

```diff
@@ -2,63 +2,88 @@
-                                          been tampered with. Typically relates to `search/scroll`
-                                          requests when the scroll ID is believed to have been
-                                          tampered with.
-| `connection_granted`              | | | Logged when an incoming TCP connection passes the
-                                          <<ip-filtering, IP Filter>> for a specific
-                                          profile.
-| `connection_denied`               | | | Logged when an incoming TCP connection does not pass the
-                                          <<ip-filtering, IP Filter>> for a specific
-                                          profile.
+When you are <<enable-audit-logging,auditing security events>>, a single client request
+might generate multiple audit events, across multiple cluster nodes. The common
+`request.id` attribute can be used to correlate the associated events.
+
+The following is an exhaustive list of the events that can be generated:
+
+|======
+| `anonymous_access_denied`         | | Logged when a request is denied due to a missing
+                                        authentication token.                                                  | | Example: `{"type":"audit", "timestamp":"2020-12-30T21:56:43,608+0200", "node.id":"0RMNyghkQYCc_gVd1G6tZQ", "event.type":"rest", "event.action":"anonymous_access_denied", "origin.type":"rest", "origin.address":"[::1]:50543", "url.path":"/twitter/_async_search", "url.query":"pretty", "request.method":"POST", "request.id":"TqA9OisyQ8WTl1ivJUV1AA"}`
+| `authentication_success`          | | Logged when a user successfully authenticates.                         | | Example: `{"type":"audit", "timestamp":"2020-12-30T22:03:35,018+0200", "node.id":"0RMNyghkQYCc_gVd1G6tZQ", "event.type":"rest", "event.action":"authentication_success", "authentication.type":"REALM", "user.name":"elastic", "user.realm":"reserved", "origin.type":"rest", "origin.address":"[::1]:51014", "realm":"reserved", "url.path":"/twitter/_search", "url.query":"pretty", "request.method":"POST", "request.id":"nHV3UMOoSiu-TaSPWCfxGg"}`
+| `authentication_failed`           | | Logged when the authentication token cannot be
+                                        matched to a known user.                                               | | Example: `{"type":"audit", "timestamp":"2020-12-30T22:10:15,510+0200", "node.id":"0RMNyghkQYCc_gVd1G6tZQ", "event.type":"rest", "event.action":"authentication_failed", "user.name":"elastic", "origin.type":"rest", "origin.address":"[::1]:51504", "url.path":"/_security/user/user1", "url.query":"pretty", "request.method":"POST", "request.id":"POv8p_qeTl2tb5xoFl0HIg"}`
+| `realm_authentication_failed`     | | Logged for every realm that fails to present a valid
+                                        authentication token. `<realm>` represents the
```

Comment:
>  `<realm>` represents the realm type. 

It's unclear where "<realm>" appears.  Is it the "realm" property in the example?

Reply:
Good spot, I don't know what that's talking about either, `realm` is the realm name. I've removed it altogether, it was a carry-over from long ago.